### ISSUE TITLE
[RTL] Handsontable cell type

### ISSUE THREAD

**Issue by Siemienik** [handsontable/handsontable#8803]
## Description

For RTL the handsontable in handsontable opens in the wrong direction (currently: from left to right, should: from right to left)
The content of nested hot instance looks as displayed correctly.

See video:

https://user-images.githubusercontent.com/2893099/135609719-a269e080-1f23-4b95-af86-19162849edd6.mp4

## Reproduction:

1. Use this [configuration](https://github.com/Siemienik/handsontable-storybooks/blob/1ab198086b3579f3fab40e48e2ddf89838add1cc/vanilla/configurations.js#L604-L640):
```js
{
    licenseKey: 'non-commercial-and-evaluation',
    data: [
        ['Tesla', 2017, 'black', 'black'],
        ['Nissan', 2018, 'blue', 'blue'],
        ['Chrysler', 2019, 'yellow', 'black'],
        ['Volvo', 2020, 'white', 'gray']
    ],
    colHeaders: ['Car', 'Year', 'Chassis color', 'Bumper color'],
    columns: [
        {
            type: 'handsontable',
            handsontable: {
                colHeaders: ['Marque', 'Country', 'Parent company'],
                autoColumnSize: true,
                data: [
                    { name: 'BMW', country: 'Germany', owner: 'Bayerische Motoren Werke AG' },
                    { name: 'Chrysler', country: 'USA', owner: 'Chrysler Group LLC' },
                    { name: 'Nissan', country: 'Japan', owner: 'Nissan Motor Company Ltd' },
                    { name: 'Suzuki', country: 'Japan', owner: 'Suzuki Motor Corporation' },
                    { name: 'Toyota', country: 'Japan', owner: 'Toyota Motor Corporation' },
                    { name: 'Volvo', country: 'Sweden', owner: 'Zhejiang Geely Holding Group' }
                ],
                getValue() {
                    const selection = this.getSelectedLast();

                    // Get the manufacturer name of the clicked row and ignore header
                    // coordinates (negative values)
                    return this.getSourceDataAtRow(Math.max(selection[0], 0)).name;
                },
            }
        },
        {},
        {},
        {}
    ]
};
```

**Comment by Siemienik** [comment]
done in #9105

**Comment by aninde** [comment]
Fixed with v12.0.0 https://jsfiddle.net/cqpvg75w/2/

**Comment by AMBudnik** [comment]
Closed with https://github.com/handsontable/handsontable/issues/8760 and developed and done in Handsontable v12.

For more information please check out our Release Notes - https://handsontable.com/docs/release-notes/#_12-0-0
